### PR TITLE
Add `enableContextMenu` description to docs.

### PR DESCRIPTION
**Pull request by m-bert**
## Description

This PR adds `enableContextMenu` property description to our documentation.

## Test plan

Run docs.

### CODE REVIEW

**Comment by kacperkapusciak** [comment]
I think this PR is more than just a "tiny change in docs" 😁

**Review by kacperkapusciak — CHANGES_REQUESTED** [review]
I think this PR is more than just a "tiny change in docs" 😁

**Comment by m-bert** [comment]
>I think this PR is more than just a "tiny change in docs" 😁

Not exactly, the thing is I've set target branch as the one with the `mouseButton` docs since those changes were connected 😅
Now after merging #2789 it should be ready to review.

File: docs/docs/components/drawer-layout.mdx [hunk 1]
```diff
@@ -100,6 +100,10 @@ Allows users to choose which mouse button should handler respond to. The enum `M
 
 Arguments can be combined using `|` operator, e.g. `mouseButton(MouseButton.LEFT | MouseButton.RIGHT)`. Default value is set to `MouseButton.LEFT`.
 
+### `enableContextMenu(value: boolean)` (**web only**)
```
Comment (kacperkapusciak) [inline docs/docs/components/drawer-layout.mdx:103]:
```suggestion
### `enableContextMenu(value: boolean)` (Web only)
```

File: docs/docs/gestures/gesture-detector.md [hunk 2]
```diff
@@ -42,6 +42,10 @@ Starting with Reanimated 2.3.0 Gesture Handler will provide a [StateManager](/do
 
 This parameter allows to specify which `userSelect` property should be applied to underlying view. Possible values are `"none" | "auto" | "text"`. Default value is set to `"none"`.
 
+### `enableContextMenu(value: boolean)` (**web only**)
```
Comment (kacperkapusciak) [inline docs/docs/gestures/gesture-detector.md:45]:
```suggestion
### `enableContextMenu(value: boolean)` (Web only)
```

**Review by kacperkapusciak — APPROVED** [review]
Consistency nit, other than that LGTM

File: docs/docs/gestures/gesture-detector.md (hunk 2 above)
Comment (m-bert) [reply docs/docs/gestures/gesture-detector.md:45]: In order to stay consistent, we will have to change it also in `userSelect` property. We could also go through our docs and change all occurrences (it appears that there are only 2 beside those mentioned here). What do you think?

Comment (kacperkapusciak) [reply docs/docs/gestures/gesture-detector.md:45]: yeah, change it everywhere to one format please

Comment (m-bert) [reply docs/docs/gestures/gesture-detector.md]: Done in [f772fba](https://github.com/software-mansion/react-native-gesture-handler/pull/2790/commits/f772fbac11af58f89f051c0e901626bd238eb1c9)

**Pull request merged**